### PR TITLE
Add ability to disable borg prune call completly

### PR DESCRIPTION
**Pull request by hashworks**
The client might not have prune permissions on the backup host.

### CODE REVIEW

File: templates/borg-backup.sh.epp [hunk 1]
```diff
@@ -160,13 +161,15 @@ backup_borg() {
   fi
 
   borg create "${options[@]}" --stats --verbose "$dst::backup-$(date "+%Y%m%d-%H%M%S")" "$src"
+<% if ($manage_prune == true) { -%>
```
Comment (bastelfreak) [inline templates/borg-backup.sh.epp:164]: you can shorten this to:
```suggestion
<% if $manage_prune { -%>
```

Comment (hashworks) [reply templates/borg-backup.sh.epp]: See https://github.com/voxpupuli/puppet-borg/pull/137/files#diff-fb441c2a865ae587ac8c8a6f2b324f1414a3821d099b31c5b5a077533ef0dad9L20, I just referenced the syntax there. Shorted them all.

**Comment by bastelfreak** [comment]
fedora is expected to fail
